src/Tribe/Asset/Select2.php

```diff
@@ -21,9 +21,8 @@ public function handle() {
 		// we know of other plugins loading a version of select2 compatible with our needs
 		// let's not queue the script twice.
 		if ( ! $this->has_script_alias( 'select2' ) ) {
-			$script_handle = $this->prefix . '-select2';
-			wp_enqueue_script( $script_handle, $path, 'jquery', '3.2' );
-			Tribe__Events__Template_Factory::add_vendor_script( $script_handle );
+			wp_enqueue_script( 'select2', $path, 'jquery', '3.2' );
+			Tribe__Events__Template_Factory::add_vendor_script( 'select2' );
```

Comment:
With this change, I wonder if we even need the wrapping if statement. That was added by @bordoni to allow us to have a prefixed version of select2.

That being said, I wonder what the merit of having our own prefixed select2 really is. It seems kind of funky because there's no case where we'd want the script enqueued twice as far as I know (unless there's a no-conflict for select2 3.x and 4.x).